homeassistant/components/honeywell/config_flow.py

```diff
@@ -83,14 +82,16 @@ async def async_step_reauth_confirm(
         return self.async_show_form(
             step_id="reauth_confirm",
             data_schema=self.add_suggested_values_to_schema(
-                REAUTH_SCHEMA, self.entry.data
+                REAUTH_SCHEMA,
+                self.entry.data,
             ),
             errors=errors,
+            description_placeholders={"name": "Honeywell"},
```

Comment:
Can we add tests for this?

Reply:
Someone else made the 1/2 completed change - modifying the standard message screwing up other integrations. I'll just leave it till they fix it.